static/js/giphy.js

```diff
@@ -33,12 +33,33 @@ export function focus_current_edit_message() {
 const APPROX_HEIGHT = 350;
 const APPROX_WIDTH = 300;
 
+export function update_giphy_rating() {
+    if (page_params.realm_giphy_rating === 0) {
+        $("#compose_box_giphy_grid").hide();
+    } else {
+        $("#compose_box_giphy_grid").show();
+    }
+}
+
+function get_rating() {
+    const options = page_params.giphy_rating_options;
+    for (const rating in page_params.giphy_rating_options) {
+        if (options[rating].id === page_params.realm_giphy_rating) {
+            return rating;
+        }
+    }
+
+    // While this is a good failsafe value, it will never be run
+    // because `giphy_rating` cannot hold any value which is not
+    // in `giphy_rating_options`.
+    return "g";
```

Comment:
Might be better to do `throw new Error` here, since it's an invalid state and we'd want to debug it?

Reply:
Added a blueslip error for this.